src/solvers/imex_ssprk.jl

```diff
@@ -144,6 +146,9 @@ function step_u!(integrator, cache::IMEXSSPRKCache)
                 T_exp!(T_exp, U, p, t_exp)
             end
         end
+        dss!(p, U_imp, t_imp) # mutate state (e.g., dss! + BCs)
```

Comment:
@dennisYatunin, should we be dss-ing the implicit part in the imex ssprk?

Reply:
No, we should not add another dss call here. For both IMEX-ARK and IMEX-SSPRK, we only need the two dss calls outlined in the [docs](https://clima.github.io/ClimaTimeSteppers.jl/dev/algorithm_formulations/ode_solvers/#IMEX-SSPRK): one when computing the residual for the implicit solve on each stage, and another at the end of the timestep.

Reply:
In both cases, `dss!` can be replaced with a generic `filter!`, since the reasoning given [here](https://clima.github.io/ClimaTimeSteppers.jl/dev/algorithm_formulations/ode_solvers/#Adding-DSS) is not specific to dss.

Reply:
Ok, I've called it `apply_filter!` (to avoid name collisions with `Base.filter!`), does that sound okay? We could alternatively adopt OrdinaryDiffEq's name `affect!`